sql/telemetry_derived/nondesktop_day_2_7_activation/query.sql

```diff
@@ -0,0 +1,61 @@
+WITH base AS (
+  SELECT
+    CASE
+      app_name
+    WHEN
+      'Fennec'
+    THEN
+      CONCAT(app_name, ' ', os)
+    WHEN
+      'Focus'
+    THEN
+      CONCAT(app_name, ' ', os)
+    WHEN
+      'Lockbox'
+    THEN
+      CONCAT('Lockwise ', os)
+    WHEN
+      'Zerda'
+    THEN
+      'Firefox Lite'
+    ELSE
+      app_name
+    END
+    AS product_name,
```

Comment:
```suggestion
    AS product,
```

We call this `product` in the exact_mau28 tables and I would like to keep it consistent unless there's a good reason to deviate.

Reply:
Change made